.github/workflows/ci.generate.ts

```diff
@@ -718,7 +738,7 @@ const ci = {
           name: "Test debug (fast)",
           if: [
             "matrix.job == 'test' && matrix.profile == 'debug' &&",
-            "!startsWith(matrix.os, 'ubuntu')",
+            "(startsWith(github.ref, 'refs/tags/') || !startsWith(matrix.os, 'ubuntu'))",
```

Comment:
We were skipping test debug on ubuntu for tag builds

Reply:
I believe this was done on purpose to limit CI run time for tags.

Reply:
It's a bit odd because we run the fast tests on every other platform, but ubuntu (the fastest runner) doesn't. It kind of feels like an accidental bit of logic.